packages/model-viewer/src/test/features/ar-spec.ts

```diff
@@ -139,6 +139,21 @@ suite('ModelViewerElementBase with ARMixin', () => {
           expect(url.hash).to.equal(
               '#custom=path-to-banner.html&allowsContentScaling=0');
         });
+
+        test('replicate src hash to usdz blob url', () => {
+          element.src = 'https://example.com/model.gltf#custom=path-to-banner.html';
+          element.arModes = 'webxr scene-viewer quick-look';
+
+          (element as any)[$openIOSARQuickLook]();
```

Comment:
The reason your test fails is that this is an async function, so you need to `await` it. It happens to work in the other tests because they aren't triggering its async nature (that was added for `prepareUSDZ`, and apparently these were not updated).